### DISCUSSION TITLE
setup release-notes-generator-action project

### DISCUSSION THREAD

**Discussion by jeevitesh** [Decathlon/release-notes-generator-action#38] [General]
Hi,
Need to setup release-notes-generator-action project. kindly share the steps, I'm not able to follow the steps mentioned in the link https://github.com/Decathlon/release-notes-generator-action#common-usage is there any other material to setup

**Comment by mmornati** [comment]
Thanks to open a discussion (instead of an issue... seems interesting way to go IMHO... I will open an Issue if the doc can be improved).

Can you provide where you are blocked in the action, or part of the logs or whatever?
I'm testing the action in this repository: https://github.com/mmornati/test-relese-notes You can find there how it is set up (it should be something like the README file).

Let me know if I can help you in some way... or let us know what is not clear in the readme... I will improve it.
